Tests/QueuesTests/Utilities/CapturingMetricsSystem.swift

```diff
@@ -0,0 +1,160 @@
+@testable import CoreMetrics
+import NIOConcurrencyHelpers
+import Vapor
+
+final class CapturingMetricsSystem: MetricsFactory {
```

Comment:
Can we use https://github.com/apple/swift-metrics/blob/main/Sources/MetricsTestKit/TestMetrics.swift instead?